client/src/components/Sider/data/menuItems.tsx

```diff
@@ -191,8 +198,8 @@ const courseManagementMenuItems: CourseManagementMenuItemsData[] = [
     courseAccess: isCourseManager,
   },
   {
-    name: 'Cross-Сheck Table',
-    key: 'Cross-Сheck Table',
+    name: 'Cross-Check Table',
```

Comment:
If you're curious what's the difference – there was one non-Latin "C" in "Cross-Check"